mini.go

```diff
@@ -100,16 +100,16 @@ func GET(writer http.ResponseWriter, reader *http.Request) {
 	//check for errors in scan  (404 and 500)
 	if err == sql.ErrNoRows {
 		fmt.Println(err)
+		writer.WriteHeader(http.StatusNotFound)
 		io.WriteString(writer, "{\"message\" : \"Error 404\"}")
```

Comment:
you can use Go's 'raw string' syntax here (delimited by backticks) to avoid escaping quotes:
```io.WriteString(writer, `{"message" : "Error 404"}`)```